distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -139,10 +139,9 @@ type DB_Table
     @selector Widget_Helpers.make_column_name_selector
     at : Integer | Text -> DB_Column ! No_Such_Column | Index_Out_Of_Bounds
     at self (selector:(Integer | Text)=0) = 
-        Feature.Column_Operations.if_supported_else_throw self.connection.dialect "at" <|
```

Comment:
Why is this check no longer neeeded for any backend?

Reply:
Similar to the removal of set. All dialects should allow getting a column from a table. What you can do with that table is then limited by the operation flagging system.